### ISSUE TITLE
Detecting short tones

### ISSUE THREAD

**Issue by dreidenbaugh** [introlab/odas#153]
I am trying to use ODAS to localize a single, short (~0.5 s) tone at a known frequency. Are there any settings I can configure that would better suit this use case? Currently, ODAS is picking up square waves like these 1000 Hz tones (around 4.8 and 6.0 s) which have multiple harmonics:
![spectrogram (square)](https://user-images.githubusercontent.com/21310897/56578185-fae21400-659a-11e9-9a00-0ce01235249b.png)

But ODAS does not recognize as a source sine waves like these (around 7.0 and 8.4 s) which are only intense at 1000 Hz:
![spectrogram (sine)](https://user-images.githubusercontent.com/21310897/56578186-fd446e00-659a-11e9-8c49-ea523463c481.png)

**Comment by FrancoisGrondin** [comment]
Yes it's because ODAS relies on GCC-PHAT, which performs well with wideband signals but not narrowband signals... Your square wave generates harmonics and thus this constitutes a larger band than your sine wave... unfortunately, so far with the actual method the system is unable to localize sine waves.
